exercises/concept/object-relational-mapping/.docs/instructions.md

```diff
@@ -10,11 +10,20 @@ Note that, internally, the database transitions through a number of state: Close
 
 The database has the following instance methods:
 
-- `Database.BeginTransaction()` starts a transaction on the database. If this is called when the database is not in a `Closed` state then an exception is thrown. If successful the internal state of the database will change to `TransactionStarted`.
-- `Database.Write(string data)` writes data to the database within the transaction. If it receives bad data an exception will be thrown. An attempt to call this method without `BeginTransaction()` having been called will cause an exception to be thrown. If successful the internal state of the database will change to `DataWritten`.
-- `Database.EndTransaction()` commits the transaction to the database. It may throw an exception if it can't close the transaction or if `Database.BeginTransaction()` had not been called.
+- `Database.Begin()` starts a transaction on the database. If this is called when the database is not in a `Closed` state then an exception is thrown. If successful the internal state of the database will change to `TransactionStarted`.
```

Comment:
Isn't the original correct though?

Reply:
Ah, true. I got confused by the method in `Orm` that is called `Begin`.